sync/course-db.js

```diff
@@ -1273,6 +1273,11 @@ module.exports.loadQuestions = async function (coursePath) {
     validate: validateQuestion,
     recursive: true,
   });
+  for (let qid in questions) {
```

Comment:
Can you add a comment explaining why we're forbidding `@...`?

Reply:
sure, it doesn't seem clear what tense it should be in though. 
> Don't allow questions to start with '@', because it will be used to reference questions outside the course.
or
> Don't allow questions to start with '@', because it is used to reference questions outside the course.

If I do the first, we have to come back and edit the comment later, if we do the second then its technically a lie until we push the rest.

Reply:
I think "will be" is fine, as that's an accurate description at this time.